src/cmdline.ggo

```diff
@@ -287,7 +287,9 @@ option "forcedns1"   - "Force all DNS to a specific address" string no
 option "forcedns1port"   - "Force all DNS to a specific port" int default="0" no 
 
 option "forcedns2"   - "Force all secondary DNS to a specific address" string no 
-option "forcedns2port"   - "Force all secondary DNS to a specific port" int default="0" no 
+option "forcedns2port"   - "Force all secondary DNS to a specific port" int default="0" no
+
+option "dnspersession"   - "Enable DNS servers per session" flag off
```

Comment:
This change, from what I can tell, is a per session force DNS. The description of "DNS servers per session" I think is misleading as people would assume you just mean per session DNS1/DNS2 values returned in DHCP.